src/Web/Masa.Auth.Web.Admin.Rcl/wwwroot/i18n/zh-CN.json

```diff
@@ -116,6 +116,8 @@
   "Team Type": "团队类型",
   "Create Team": "创建团队",
   "Update Team": "编辑团队",
+  "Delete Team": "删除团队",
+  "Confirm Delete Team": "确认删除该团队？",
```

Comment:
Is this a sentence or some words? Why some words like `Confirm Delete Team` and some words like `Failed to add menuPermission`?